### PR TITLE
v1.16 Docs

### PR DESCRIPTION
**Pull request by tr0njavolta**
https://github.com/crossplane/docs/issues/772

Updates master with version 1.16

- [x] Update the $LATEST_VER parameter in [netlify_build.sh](https://github.com/crossplane/docs/blob/master/netlify_build.sh#L3)
- [x] Update params.latest in [config.yaml](https://github.com/crossplane/docs/blob/master/config.yaml#L93)
- [x] Update version in the _index.md file of /content/<new latest> from master to the correct version.
- [x] Copy Crossplane [cluster/crds](https://github.com/crossplane/crossplane/tree/master/cluster/crds) contents to /content/<new latest>/api/crds.
- [x] Create a [new release/tag](https://github.com/crossplane/docs/releases/new) named "v-archive" to snapshot EOL'd docs.
- [x] Remove EOL'd docs version from "/content" directory and run hugo locally to check for broken links.
- [ ] Trigger [Algolia Crawler](https://crawler.algolia.com/) after publishing to reindex results.

### CODE REVIEW

**Review by jbw976 — COMMENTED** [review]
Awesome thank you for taking this on @tr0njavolta! Does this address everything in https://github.com/crossplane/docs/issues/772, or a subset of that checklist?  I'll start taking a deeper pass through now, including checking out the preview site to make sure functionality looks to be in the right place 😉

File: content/master/guides/import-existing-resources.md
Comment (jbw976) [inline content/master/guides/import-existing-resources.md:8]: should this be set to `/latest/...` instead? do we have precedence for that with links in other places on this docs site?

**Comment by tr0njavolta** [comment]
> Awesome thank you for taking this on @tr0njavolta! Does this address everything in #772, or a subset of that checklist? I'll start taking a deeper pass through now, including checking out the preview site to make sure functionality looks to be in the right place 😉

@jbw976 updated with the checklist and verified with the preview. API/CRD updates look good and version changes are in the right place.

The Vale issues I can tackle in a separate PR because that content was already present so there's probably a rule not applied in the new version that I can check out.

**Comment by jbw976** [comment]
> Create a [new release/tag](https://github.com/crossplane/docs/releases/new) named "v-archive" to snapshot EOL'd docs.

I don't see the `v1.13-archive` release/tag yet - do you? 🤔

File: content/v1.13/concepts/composition-functions.md
Comment (jbw976) [inline content/v1.13/concepts/composition-functions.md:1]: https://github.com/crossplane/docs/tree/master/content/v1.13 has more files than what look to have been deleted in this PR - is there more to delete here still?

File: content/v1.15/guides/import-existing-resources.md
Comment (jbw976) [inline content/v1.15/guides/import-existing-resources.md:8]: should this be pointing to v1.15 as its in the v1.15 docs?

File: content/v1.15/guides/import-existing-resources.md
Comment (jbw976) [inline content/v1.15/guides/import-existing-resources.md:87]: same, should this be 1.15?

**Review by jbw976 — COMMENTED** [review]
Looks mostly pretty good here @tr0njavolta!! a couple questions included :)

File: content/v1.13/concepts/composition-functions.md
Comment (tr0njavolta) [reply content/v1.13/concepts/composition-functions.md:1]: It seems that the content from v1.13 that matched the content coming from master > v1.16 came out as a wash to git.

File: content/master/guides/import-existing-resources.md
Comment (tr0njavolta) [reply content/master/guides/import-existing-resources.md:8]: previously it was pinned to the version number so I went with that.

Comment (jbw976) [reply content/master/guides/import-existing-resources.md:8]: fair enough - let's not rock the boat too much on this release then 😂

File: content/v1.13/concepts/composition-functions.md
Comment (jbw976) [reply content/v1.13/concepts/composition-functions.md:1]: oh interesting you're right, looks like there is no v1.13 folder in your fork: https://github.com/tr0njavolta/xp-docs-fork/tree/master/content

**Comment by jbw976** [comment]
Looks like the latest commit didn't have a sign-off, so the DCO check is failing: https://github.com/crossplane/docs/pull/773/checks?check_run_id=25023037420

Can you fix that up please? 🙏  this link may help: https://github.com/crossplane/crossplane/tree/master/contributing#certificate-of-origin

**Comment by jbw976** [comment]
> I don't see the v1.13-archive release/tag yet - do you? 🤔

This was a blocker for @tr0njavolta because they didn't have permissions to create a new release/tag in the Github UI. I created the release/tag at https://github.com/crossplane/docs/releases/tag/v1.13-archive

**Review by jbw976 — APPROVED** [review]
Awesome, I think we've resolved everything in this PR now, looks good to me @tr0njavolta! thank you for your work on this docs release!!

I'll approve now, but then merge this when I'm ready with the rest of the v1.16 release process so they stay in sync: https://github.com/crossplane/release/issues/9

**Pull request merged**
